### DISCUSSION TITLE
Unification of storyline of KodaDot

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#2952] [General]
Would be amazing to have one unified storyline of KodaDot which would spread much faster and helped the ambassadors to shill the KodaDot brand much easier and replicate it through various audiences way better.

The advantage of this is we can add quality to our storyline and control distribution is much better among believers :)

### My take 
- KodaDot is the NFT Marketplace developed as a public good of the Dotsama ecosystem. 
- It's developed in web3 native way, where public contributors are paid right away for their contribution in form of bounties.
- Whole development is made in a way as transparent and open development, in an async-first fashion
- I can add more later 

🙌 Feel free to add your sauce what's your perspective on KodaDot

**Comment by DeluneClaire** [comment]
In my opinion one of the things i like the most or i I appreciate is the fact we do something about nature. Its beautiful and i think that should be some thing we definetely mention as much as we can. I tried to write something shorter than we have on Koda-web but its really very complex text with many details informantion. 

- With big name comes even bigger responsibility and we feel like we should step up in the game and be more carbon neutral and later contribute to be carbon negative gallery in a way to drive global forestation.  
- We are running on a low carbon blockchain Kusama network, where artists can offset their mints. Carbon Neutral Mission of Kodadot.
- To kick-start this effort, we've partnered with Offsetra which allows users to support "high-quality pro-climate projects via the purchase of carbon offsets credits".


Just let me know if it makes sanse when you read it 😁
